Code/Engine/Foundation/Communication/Message.h

```diff
@@ -50,7 +50,8 @@ class EZ_FOUNDATION_DLL ezMessage : public ezReflectedClass
   /// \brief Calculates a hash of the message.
   EZ_ALWAYS_INLINE ezUInt32 GetHash() const
   {
-    return ezHashingUtils::xxHash32(this, m_uiSize);
```

Comment:
Why?

Reply:
Change was originally requested by clemens as small performance optimization. After a brief discussion decided to revert.